scrapers/MetArtNetwork.yml

```diff
@@ -0,0 +1,27 @@
+name: "MetArtNetwork"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - metartnetwork.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $info: //div[@class="pull-left custom-photo-movie-description"]
+      $credits: //div[@class="pull-left custom-credits-left"]
+      $script: //script[contains(text(),'INITIAL_STATE')]/text()
+    scene:
+      Title: $info/h3/a/@title
+      Details: $info//p/text()
```

Comment:
The details gets more results than it should and it only keeps the short description not the long one.
`https://www.metartnetwork.com/model/mary-moody-and-naomi-swann/movie/20200619/NO_REGRETS/`
`https://www.metartnetwork.com/model/lil-karla/movie/20200618/FORCED_ORGASM_2/`
The below works as it should at least for the above scenes
```yamlDetails: 
        selector: //div[@class="custom-description-long custom-description-hide"]/p/text()
        concat: "\n"
        replace:
          - regex: "\n"
            with: "\n" 
```
the extra replace is used to protect the "\n" as its usage is still troublesome